src/index.html

```diff
@@ -13,20 +13,32 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="style.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. Browsers can't interpret SCSS files. You need to compile your SCSS to CSS first.